### PR TITLE
docs: docs for aa-ethers

### PR DESCRIPTION
**Pull request by avasisht23**
https://github.com/OMGWINNING/aa-sdk-staging/assets/43521356/9d74499b-163e-46b1-8b6c-d9ec9eaffebb

### CODE REVIEW

**Comment by avasisht23** [comment]
Current dependencies on/for this PR:
* main
  * **PR #47** <a href="https://app.graphite.dev/github/pr/OMGWINNING/aa-sdk-staging/47" target="_blank">Graphite</a>  👈

This comment was auto-generated by [Graphite](https://app.graphite.dev/github/pr/OMGWINNING/aa-sdk-staging/47?utm_source=stack-comment).

File: site/packages/aa-ethers/provider-adapter/connectToAccount.md [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+
+`connectToAccount` is a method on `EthersProviderAdapter` that you can optionally call to connect the Provider to an Account and returns a Signer. This enables the returned `AccountSigner` to leverage the provider when signing messages, UserOperations, and transactions for a smart contract account using the owner account.
+Ø
+
+## Usage
+
+::: code-group
+
+<<< @/snippets/ethers-signer.ts
+<<< @/snippets/ethers-provider.ts
```
Comment (avasisht23) [inline site/packages/aa-ethers/provider-adapter/connectToAccount.md:25]: these snippets are sufficient as an example for this method too

File: site/packages/aa-ethers/provider-adapter/fromEthersProvider.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+
+# EthersProviderAdapter
+
+`fromEthersProvider` is a static method on `EthersProviderAdapter` that converts an ethers.js `JsonRpcProvider` to an `EthersProviderAdapter`.
+
+## Usage
+
+::: code-group
+
+<<< @/snippets/ethers-provider.ts
```
Comment (avasisht23) [inline site/packages/aa-ethers/provider-adapter/fromEthersProvider.md:23]: this snippet is also a sufficient example for this method

**Review by denniswon — APPROVED** [review]
LGTM. just nit comments made cc: @avasisht23

File: site/packages/aa-ethers/account-signer/connect.md [hunk 3]
```diff
@@ -0,0 +1,54 @@
+---
+outline: deep
+head:
+  - - meta
+    - property: og:title
+      content: AccountSigner • connect
```
[the remark this answers is not in the document]
Comment (avasisht23) [reply site/packages/aa-ethers/account-signer/connect.md:6]: Ah yeah the format I used is what Moldy recommended me on a prev PR

File: site/snippets/ethers-provider.ts [hunk 4]
```diff
@@ -0,0 +1,15 @@
+import { EthersProviderAdapter } from "@alchemy/aa-ethers";
+import { Alchemy, Network } from "alchemy-sdk";
+
+export const entryPointAddress = "0x5FF137D4b0FDCD49DcA30c7CF57E578a026d2789";
+
+const alchemy = new Alchemy({
+  apiKey: process.env.API_KEY!,
+  network: Network.MATIC_MUMBAI,
+});
+const alchemyProvider = await alchemy.config.getProvider();
```
[the remark this answers is not in the document]
Comment (avasisht23) [reply site/snippets/ethers-provider.ts:10]: Good call

**Pull request merged**